_layouts/landing.html

```diff
@@ -38,7 +38,7 @@
 
 
       <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-151519261-1"></script>
+    <script async rel="preconnect" src="https://www.googletagmanager.com/gtag/js?id=UA-151519261-1"></script>
```

Comment:
Shouldn't the preconnect happen on a `link`? Even so, it might not be needed here as we don't mind this resource being loaded with low priority. I think the `async` is sufficient here.